src/example.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is the example code I sketched out for you, no?  Don't add this to your repository.  Or if you do, attribute it properly.